Jenkinsfile

```diff
@@ -1,4 +1,6 @@
-// All non-runtime dependencies installed in Dockerfile.ci
+#!groovy
```

Comment:
it looks like you are using jenkins only for deployment, what about running tests? making sure the typescript code is compiled? prettier  is passing? etc ..

Reply:
I will add those in a later stage.

The tests are just going to be making sure that typescript code is compiles as there is no good way that I could find for making sure that the native code is being used correctly.

What we are going to do for e2e tests is have this run in automation.

Reply:
not sure why not to add check compilation check, but ok..